cf/i18n/resources/de-de.all.json

```diff
@@ -2325,7 +2325,7 @@
   },
   {
     "id": "HEALTH_CHECK_TYPE",
-    "translation": "HEALTH_CHECK_TYPE (TYP DER ZUSTANDSPRÜFUNG)"
+    "translation": "TYP DER ZUSTANDSPRÜFUNG"
```

Comment:
In English usage we use underscores so it's clear it refers to a single input value, not multiple space-separated values (e.g. `cf push [-u TYP-DER-ZUSTANDSPRÜFUNG]` to be consistent with the use of dashes in other German translations)